collector/pg_stat_checkpointer.go

```diff
@@ -104,8 +107,15 @@ var (
 	FROM pg_stat_checkpointer;`
 )
 
-func (PGStatCheckpointerCollector) Update(ctx context.Context, instance *instance, ch chan<- prometheus.Metric) error {
+func (c PGStatCheckpointerCollector) Update(ctx context.Context, instance *instance, ch chan<- prometheus.Metric) error {
 	db := instance.getDB()
+
+	before17 := instance.version.LT(semver.MustParse("17.0.0"))
```

Comment:
This line is causing the tests to fail because the instance version is not defined. See https://github.com/prometheus-community/postgres_exporter/blob/master/collector/pg_stat_statements_test.go#L33

Reply:
ping @michael-todorovic thank you!

Reply:
@sysadmind it should be good now. thank you!